src/main/kotlin/com/craftschemers/hub/HubPlayer.kt

```diff
@@ -1,8 +1,17 @@
 package com.craftschemers.hub
 
 import com.craftschemers.hub.minigame.GameType
+import com.craftschemers.hub.minigame.sendErrorMessage
 import org.bukkit.entity.Player
 
-class HubPlayer(val player: Player) {
+data class HubPlayer(val player: Player) {
     var game: GameType? = null
+
+    fun sendMessage(message: String) {
+        player.sendMessage(message)
+    }
+
+    fun sendErrorMessage(message: String) {
+        player.sendErrorMessage(message)
+    }
```

Comment:
Since this is just calling extension functions on a property we should just make these extension functions as well or just make the call on the property, it's public so you should be able to just do `hubPlayer.player.sendMessage`.